packages/angular/ssr/node/src/app-engine.ts

```diff
@@ -22,6 +23,10 @@ import { createWebRequestFromNodeRequest } from './request';
 export class AngularNodeAppEngine {
   private readonly angularAppEngine = new AngularAppEngine();
 
+  constructor() {
+    attachNodeGlobalErrorHandlers();
```

Comment:
As an alterative, we can export this a public API, and let the users invoke it manually in the `server.ts`